DESCRIPTION

```diff
@@ -5,6 +5,8 @@ Title: Deploy Docs, Apps, and APIs to Posit Connect, shinyapps.io, and
 Version: 0.8.29-1
 Authors@R: c(
     person("Aron", "Atkins", , "aron@posit.co", role = c("aut", "cre")),
+    person("Toph", "Allen", , "toph@posit.co", role = "aut"),
+    person("Hadley", "Wickham", , "hadley@posit.co", role = "aut"),
```

Comment:
FWIW we normally only supply the email address for the maintainer, since that's the only place where it's required.

Reply:
Got it. I was following the existing local style. Jonathan left both our email addresses when he added me as maintainer; I'll strip the others.

Reply:
https://github.com/rstudio/rsconnect/commit/7feafbf1fd92e7511aefc3b6b870faa1a4d22c03, for reference.